dashboard/app/controllers/levels_controller.rb

```diff
@@ -8,7 +8,8 @@ class LevelsController < ApplicationController
   include LevelsHelper
   include ActiveSupport::Inflector
   before_action :authenticate_user!, except: [:show, :embed_level, :get_rubric]
-  before_action :require_levelbuilder_mode, except: [:show, :embed_level, :get_rubric]
+  before_action :require_levelbuilder_mode, except: [:show, :embed_level, :get_rubric, :get_filters, :get_filtered_levels]
+  before_action :require_levelbuilder_mode_or_test_env, only: [:get_filters, :get_filtered_levels]
```

Comment:
I just removed :get_filters from https://github.com/code-dot-org/code-dot-org/pull/37302 as I went to passing down the information directly.

Reply:
I'll wait for #37302 as well.

Reply:
All set!